1.11/installing/ent/custom/advanced/index.md

```diff
@@ -187,60 +202,70 @@ To install DC/OS:
     
     At this point your directory structure should resemble:
     
-        ├── dcos-genconf.c9722490f11019b692-cb6b6ea66f696912b0.tar
-        ├── dcos_generate_config.ee.sh
-        ├── genconf
-        │   ├── config.yaml
-        │   ├── ip-detect
-             
+    ```
+    ├── dcos-genconf.c9722490f11019b692-cb6b6ea66f696912b0.tar
+    ├── dcos_generate_config.ee.sh
+    ├── genconf
+    │   ├── config.yaml
+    │   ├── ip-detect
```

Comment:
add the `license.txt` in this tree

Reply:
What does "add the license.txt in this tree" mean?

Reply:
out license.txt into the tree below ip-detect